src/ImageSharp/Formats/Jpeg/JpegConstants.cs

```diff
@@ -173,6 +178,46 @@ internal static class Markers
             /// </summary>
             public const byte SOF2 = 0xC2;
 
+            /// <summary>
+            /// Start of Frame marker, non differential lossless, Huffman coding.
+            /// </summary>
+            public const byte SOF3 = 0xC3;
+
```

Comment:
note: there are also 0xC5 and 0xC6 StartOfFrame markers for Differential progressive DCT and Differential sequential DCT. I could not find any real life examples of those, so I was unsure if I should add those here too.

Reply:
Afaik arithmetic coding is not very common (at least for general internet use case) as it's harder to compute and storage gain is not really better than DCT one. Adding this just for 'why-not-ness' won't break anything as it's internal or kill the performance. So, why not?

Reply:
If anyone gets bored they could copy implementations of the missing decoder functionality from here. 

https://github.com/yigolden/JpegLibrary

They copied a bunch of stuff from us so all fair.